plotting/plot_shapes.py

```diff
@@ -92,12 +97,16 @@ def main(args):
             "misc": "misc",
             "noniso": "noniso"
         }
-
-    if args.normalize_by_bin_width:
-        split_value = 10001
+    if args.log == True:
+        if args.normalize_by_bin_width:
+            split_value = 10001
+        else:
+            split_value = 101
     else:
-        split_value = 101
+        split_value = 0
     split_dict = {c: split_value for c in ["et", "mt", "tt"]}
+    if args.xlabeling is not None:
+        xlabeling = yaml.load(open(args.xlabeling))
```

Comment:
Currently the yaml file would be included directly here and via styles.py. I'd prefer the yaml to be only included in styles.py and all helper dicts as for labels and colors accessed this way.